include/deal.II/grid/reference_cell.h

```diff
@@ -1140,6 +1140,8 @@ ReferenceCell::face_to_cell_lines(const unsigned int  face,
   AssertIndexRange(face, n_faces());
   AssertIndexRange(line, face_reference_cell(face).n_lines());
```

Comment:
If you want, move the changes to this file into a separate PR. They're logically independent.